config/routes.rb

```diff
@@ -6,12 +6,17 @@
   # You can have the root of your site routed with "root"
   root 'home#show'
 
+  require 'sidekiq/web'
+  require 'sidekiq_constraint'
+  Sidekiq::Web.set :session_secret, Rails.application.secrets[:secret_key_base]
```

Comment:
It seems to be what the sidekiq folks suggest (setting the session_secret in the routes).  I'm guessing because it has something to do with how the context in which the rack app is loaded and mounted vs. how the parent rails app is initialized (which is where `config/initializers` comes into play).  I think we shoulkd keep it as-is.